File: RxJavaRetrofit/app/src/main/java/com/woowrale/rxjavaretrofit/data/repository/ApiServices.java
package com.woowrale.rxjavaretrofit.data.repository;

import com.woowrale.rxjavaretrofit.data.model.Crypto;
import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface ApiServices {

    @GET("{coin}-usd")
    Observable<Crypto> getCoinData(@Path("coin") String coin);
}

File: RxKotlinRetrofit/app/src/main/java/com/woowrale/rxkotlinretrofit/data/repository/ApiServices.kt
package com.woowrale.rxkotlinretrofit.data.repository

import com.woowrale.rxkotlinretrofit.BuildConfig
import com.woowrale.rxkotlinretrofit.data.model.Crypto
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Path

interface ApiServices {

    @GET("{coin}-usd")
    fun getCoinData(@Path("coin") coin: String): Observable<Crypto>

    companion object {
        val BASE_URL = BuildConfig.BASE_URL
    }
}